package.json

```diff
@@ -83,7 +87,14 @@
     "tailwindcss-animate": "^1.0.6",
     "tiny-invariant": "^1.3.1",
     "ua-parser-js": "^1.0.35",
-    "zod": "^3.21.4"
+    "zod": "^3.21.4",
+    "dotenv": "^16.3.1",
```

Comment:
Not sure why we need dotenv in dependencies. We already have it in the devDependencies.